net-misc/sysrepo/sysrepo-1.4.104.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="YANG-based configuration and operational state data store"
+HOMEPAGE="https://www.sysrepo.org"
+SRC_URI="https://github.com/sysrepo/sysrepo/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+# tests trigger sandbox violation
+RESTRICT="test"
```

Comment:
any chance of fixing this?

Reply:
Oh it seems to be working now :smiley:  I could swear last time I was trying it out there were some nasty sandbox issues. Might be something with cmake-multilib eclass I was using before